paper-fab.html

```diff
@@ -165,7 +154,30 @@
       mini: {
         type: Boolean,
         value: false
+      },
+
+      /**
+       * If true, the button should be styled with a shadow.
+       */
+      raised: {
```

Comment:
I thought they were raised (i.e. not flat) by default. No? Does it have a different meaning?